CHANGELOG.md

```diff
@@ -9,9 +9,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - Fixed a typo in the Python type stubs for `ferveo.Keypair.secure_randomness_size()`. ([#61])
+- Replaced raw tuples with `ValidatorMessage` in Python bindings. ([ferveo#131])
```

Comment:
I believe this is a breaking change, i.e. major version change...? 
Also, should this just link to this PR (#65) instead of the ferveo link? I think the goal is to link to `nucypher-core` specific changes.

(^ cc @fjarri )

Reply:
>  i.e. major version change...?

I think your changelog entries should be under `0.10.0` because of the breaking change.

Reply:
They should be under `### Changed` subsection, not `### Fixed`.